api/base/caller.go

```diff
@@ -15,7 +15,7 @@ import (
 	"gopkg.in/macaroon.v2"
 )
 
-//go:generate mockgen -package mocks -destination mocks/caller_mock.go github.com/juju/juju/api/base APICaller,FacadeCaller
+//go:generate go run github.com/golang/mock/mockgen -package mocks -destination mocks/caller_mock.go github.com/juju/juju/api/base APICaller,FacadeCaller
```

Comment:
Why do we need this rather than just using mockgen assuming it was built?

Reply:
Because mockgen generates code that is checked in, we should all be using the same version. By using go run we force go to use the go.mod locked version.